scripts/processFilters.py

```diff
@@ -1,4 +1,11 @@
-
+####################################################################################################
+# Script name : processFilters.py
+#
+# fit the band filters with a gaussian mixture
+# if make_plot, save images
+#
+# output file : band + '_gaussian_coefficients.txt'
+#####################################################################################################
```

Comment:
why is there a code fitting band filters with a gaussian mixtures here and in interface/rails?